stripe/src/main/java/com/stripe/android/view/AddPaymentMethodActivity.kt

```diff
@@ -75,11 +79,21 @@ open class AddPaymentMethodActivity : StripeActivity() {
 
     private fun configureView(args: AddPaymentMethodActivityStarter.Args) {
         viewStub.layoutResource = R.layout.add_payment_method_layout
-        val contentRoot = viewStub.inflate() as ViewGroup
+        val scrollView = viewStub.inflate() as ViewGroup
+        val contentRoot = scrollView.getChildAt(0) as ViewGroup
```

Comment:
let's set an ID on the view and use findViewById instead of referencing it by child position, in case its position ever changes

Reply:
👍